packages/core/src/components/Form.tsx

```diff
@@ -112,7 +112,7 @@ export interface FormProps<T = any, S extends StrictRJSFSchema = RJSFSchema, F e
   onFocus?: (id: string, data: any) => void;
   // <form /> HTML attributes
   /** The value of this prop will be passed to the `accept-charset` HTML attribute on the form */
-  acceptcharset?: string;
+  acceptCharset?: string;
```

Comment:
For backwards compatibility we'll need to keep both for now:
```suggestion
/** The value of this prop will be passed to the `accept-charset` HTML attribute on the form
 * @deprecated replaced with `acceptCharset` which will supercede this value if both are specified
 */
  acceptcharset?: string;
/** The value of this prop will be passed to the `accept-charset` HTML attribute on the form */
  acceptCharset?: string;
```

Reply:
I don't see this change applied yet

Reply:
Sorry, I missed it. I applied it.